src/routes/safe/components/Balances/SendModal/screens/SendCollectible/style.ts

```diff
@@ -14,3 +14,31 @@ export const styles = createStyles({
     gap: '16px',
   },
 })
+
+export const selectedTokenStyles = createStyles({
```

Comment:
Please see my comment further up about styling in MUI.

Reply:
This also already existed. I moved it into a new file since it was duplicated before for `SendFunds` and `SendCollectible`. We should do the refactor to styled-components in a separate task.